src/main/java/com/spm/vasylyshyn/enums/CType.java

```diff
@@ -1,5 +1,5 @@
 package com.spm.vasylyshyn.enums;
 
 public enum CType {
-    GAS,WATER,LIGHT
+    GAS,WATER,LIGHT,NOT_SELECTED
```

Comment:
That's a good way to handle invalid enum values. I use `None` as the first member wherever my enum might not be initialized to a correct value, as a default. Also, we might not need it, if we should not have an ability to create a CType without a valid value, so I recoomend to review all the places that enum might be created. If we deserialize it, and it's null - the whole model should not be valid, and might be better null.